### PR TITLE
Document direnv

### PR DESCRIPTION
**Pull request by rpdelaney**
I added a doc illustrating how to use `direnv`.

I'm open to suggestions on where to put this if other folks have stronger opinions. The `/developing/` topic directory table of contents is mostly sub-topics, but "Docker" has its own page. I thought of putting this under a "tools" sub-topic but that seemed a bit generic.

### CODE REVIEW

File: developing/direnv/README.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+## Installation
+
+Via homebrew:
+
+     brew install direnv
+
+## Usage
+
+Create an `.envrc` file in a directory with some environment variables your project needs:
```
Comment (sojeri) [inline developing/direnv/README.md]: might be worth adding something re: putting this in the global gitignore

if we already have that explained under source control, then you can point back to there? :)

Comment (rpdelaney) [reply developing/direnv/README.md]: Hmmm. Sometimes the `.envrc` has things that should be source controlled though. On MilMove they did something pretty neat to resolve that tension so I can put something in about it.

Comment (rpdelaney) [reply developing/direnv/README.md]: @sojeri does 48ef255 help?

Comment (sojeri) [reply developing/direnv/README.md]: sure, that addresses the issue enough for me :)

**Comment by sojeri** [comment]
you could go for command line tools if you want to move this somewhere, but I think until we have a lot of files in here we don't need to worry about this

**Pull request merged**